vMenu/menus/OnlinePlayers.cs

```diff
@@ -47,6 +50,17 @@ private void CreateMenu()
             MenuItem ban = new MenuItem("~r~Ban Player Permanently", "Ban this player permanently from the server. Are you sure you want to do this? You can specify the ban reason after clicking this button.");
             MenuItem tempban = new MenuItem("~r~Ban Player Temporarily", "Give this player a tempban of up to 30 days (max). You can specify duration and ban reason after clicking this button.");
 
+            menu.OnMenuClose += async (sender) =>
+            {
+                await BaseScript.Delay(100);
```

Comment:
I'm open to suggestion on how to better do this.

Basically, I need to stop the timer when the menu closes and since I believe this event is called in between when main menu and Online players switches visibility, I added a small delay.